scripts/dabl_deploy.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+# Terminate on any failure
+set -ex
+
+STAMP=$(date "+%H%M%S")
+PROJECT_NAME=Cbdc$STAMP
+
+LEDGER_NAME=cbdc
+# The 'jq -r' removes the superfluous quotes for our variables.
```

Comment:
```suggestion
```

Reply:
I would rather leave this, this is the only comment helpful for me.

Reply:
doesn't `jq -h` or `man jq` gives you the same info?
we shouldn't include jq doc here imho

Reply:
if you really need it, perhaps:
```
alias jq_rawstring='jq -r'
```
and use it everywhere.
still sounds a bit redundant, but acceptable.

Reply:
removed the comment